### ISSUE TITLE
Should PatchOperationParameterConverter have an implementation for WriteJson method?

### ISSUE THREAD

**Issue by omelianlevkovych** [simpleidserver/SimpleIdServer#553]
We have **[PatchOperationParameterConverter](https://github.com/simpleidserver/SimpleIdServer/blob/master/src/Scim/SimpleIdServer.Scim/Infrastructure/Converters/PatchOperationParameterConverter.cs)** which is used for example in **PatchOperationParameter** for Patch requests.
There is no implementation for the **WriteJson** method in the **PatchOperationParameterConverter**.
Does it make sense to call the default writer from base **JsonConverter** or at least throw **NotImplementedException**?

Because if we want to log the request we will always miss the operation data and it could be not obvious why this is actually happening.

Something similar to this:
```
public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
{
    writer.WriteValue(value.ToString());
}
```

**Comment by simpleidserver** [comment]
First thank you for reporting this issue :)
It will be fixed for the next release !

KR,

SID

**Comment by omelianlevkovych** [comment]
I would like to contribute and create PR if it is possible.

**Comment by simpleidserver** [comment]
I would be pleased if you could contribute to the project :)

Instead of overriding the WriteJson method, you can make the following changes:

* Remove the `WriteJson` procedure.
* Override the property `CanWrite` like this: `public override bool CanWrite => false`.

The same problem exists in the class `RepresentationParameterConverter`, and you can apply the same changes :)

Please feel free to create a pull request on the branch `release/4.0.2`.

**Comment by simpleidserver** [comment]
The issue should be fixed in the branch `release/4.0.2` :)
Changeset : https://github.com/simpleidserver/SimpleIdServer/commit/2ebe23e2d1071881f519920033e132dce1f280e6
